sources/packages/backend/libs/sims-db/src/entities/user-types.enum.ts

```diff
@@ -0,0 +1,8 @@
+export enum InstitutionUserTypes {
+  admin = "admin",
```

Comment:
Shouldn't we use capital for enums?

Reply:
it's not pat of the PR. it was moved to libs/ folder